app/graphql/types/topic_type.rb

```diff
@@ -0,0 +1,13 @@
+module Types
+  class TopicType < Types::BaseObject
+    field :id, ID, null: false
+    field :description, String, null: false
+    field :translations, [Types::TopicTranslationType], null: true
+    field :created_at, GraphQL::Types::ISO8601DateTime, null: false
+    field :updated_at, GraphQL::Types::ISO8601DateTime, null: false
+
+    def translations
+      object.translations_for_languages(context[:language_ids])
+    end
```

Comment:
Is this where the specific translations (based on the two ids) are being set?

Reply:
Yes, exactly. But I just pushed up a refactor that instead of setting here, sets in the query and adds to context. What do you think?

Reply:
I think it's easier to read with the refactor but maybe more it was more scalable before? I'll merge this and if we need to change it later to DRY up code then let's do that